app/styles/custom/_globals.scss

```diff
@@ -116,7 +116,7 @@ input[type=radio]:focus
     &.has-icon > .dropdown-header {
```

Comment:
Not sure what  `_globals.scss` mean or rather if `.dropdown-header` should belong to `_globals.scss`. If we have 1 dedicated scss file per component then wondering if this should be in something like `_dropdown.scss`.